CONTRIBUTING.md

```diff
@@ -39,9 +39,20 @@ Guidelines for rosdep rules
 ---------------------------
 
  * Native packages are strongly preferred (They are required for packaging and have upgrade and conflict tracking.)
- * Python packages should go in the python.yaml
- * Homebrew into homebrew.yaml
+ * Please use the smallest notation possible and if something is default or not necessary to specify, don't.
```

Comment:
I don't think what mean is clear here. The terms `notation` and `is default` aren't well defined imo. Maybe you should include examples of each.